examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1537 @@
+      " 'driver_id': [1004, 1005]}\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "09/24/2024 06:04:33 PM root WARNING: _list_feature_views will make breaking changes. Please use _list_batch_feature_views instead. _list_feature_views will behave like _list_all_feature_views in the future.\n",
+      "09/24/2024 06:04:33 PM root WARNING: Cannot use sqlite_vec for vector search\n"
+     ]
+    }
+   ],
+   "source": [
+    "online_features_stores_client = online_feature_store_client.get_online_features(\n",
+    "    features=driver_activity_v4,\n",
+    "    entity_rows=[\n",
+    "        # {join_key: entity_value}\n",
+    "        {\"driver_id\": 1004},\n",
+    "        {\"driver_id\": 1005},\n",
+    "    ],\n",
+    ").to_dict()\n",
+    "pprint(online_features_stores_client)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bf1ad00-cd37-4feb-bd5d-4b00854e34dd",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Accessing Feast Registry metadata using remote registry store\n",
+    "Registry is going to have all the metadata information of feast objects such as FeatureService, FeatureViews. Either you can directly access this information using the way referred in above section.\n",
+    "\n",
+    "The other way to access in the client server model. You can start the registry server and access them using remote registry client as shown in this section.\n",
+    "\n",
+    "The default port for the registry is `6570`\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2e068e2-e3cd-4444-81b3-f335d27d47a8",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Starting the registry server as remote"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a401abd1-c250-46a9-9ed1-3f4c3b705cec",
+   "metadata": {},
+   "source": [
+    "Change the current directory context to initial feature store so that we can start the registry server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "id": "be7cf013-33db-487a-97cb-48d3b23924e1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/opt/app-root/src/feast/examples/rhoai-quickstart/my_feast_project/feature_repo\n"
+     ]
+    }
+   ],
+   "source": [
+    "%cd ./../my_feast_project/feature_repo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "835c26e8-1df9-4b02-967a-08eb9747c258",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Registry server started on the default port 6570. Go to next cell and check if the process is available.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import subprocess\n",
+    "\n",
+    "# Run feast serve in the background\n",
+    "feast_remote_registry_server_process = subprocess.Popen([\"feast\", \"serve_registry\"])\n",
```

Comment:
This cell didn't run for me. I got a "ModuleNotFound" error for grpc. Once I ran `pip install grpcio`, then it worked. This probably needs to be added to the top of the notebook.

Reply:
I am a bit surprised why you had to install it separately. I assumed it will be part of feast dependencies but anyway added this dependency now. PTAL.